great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -0,0 +1,17 @@
+from great_expectations.core.config_peer import ConfigPeer
+from great_expectations.data_context.types.base import BaseYamlConfig
+
+
+class AbstractDataContext(ConfigPeer):
```

Comment:
Consider removing `ConfigPeer` from this class and add it to the list of inherited types for DataContext and BaseDataContext